debian/patches/no-single-process.patch

```diff
@@ -247,3 +250,70 @@ Last-Update: 2024-08-02
              /dev/sdb1	none	swap	sw,comment=cloudconfig	0	0
              """  # noqa: E501
              ).strip()
+--- a/systemd/cloud-init-network.service.tmpl
```

Comment:
I know this file was inadvertently added previously and needs to be removed...but do you know why it is showing up here? I thought we had already removed it in a previous patch update.

Reply:
I think that was fixed in 24.4.x but not this branch